themes/enfocado_neon_dark.toml

```diff
@@ -0,0 +1,34 @@
+# Theme: enfocado_neon_dark
+# Source:  https://github.com/wuelnerdotexe/vim-enfocado
+
+# Default colors
+[colors.primary]
+background = '#181818'
+foreground = '#f8f8f2'
+
+# uncomment the following tow lines to color cursor
+[colors.cursor]
+# text   = '#181818'
+# cursor = '#faffea'
+
+# Normal colors
+[colors.normal]
+black   = '#4a4a4a'
+red     = '#ed4a46'
+green   = '#70b433'
+yellow  = '#ee5400'
+blue    = '#a572d2'
+magenta = '#eb4eb7'
+cyan    = '#56b8c9'
```

Comment:
How did you arrive at these colors? They do not match the enfocado theme source.

Reply:
The goal of these shared themes is to give people easy access to popular themes. If you modify themes to be your personal ideal colorscheme instead then the potential userbase for that is exactly 1 person, at which point there's no reason for us to host that theme.

So generally speaking the original colors should be used, so when someone interested in the enfocado theme looks through this repository, it's actually what they get.